miprometheus/models/vision/lenet5.py

```diff
@@ -48,11 +48,11 @@ def __init__(self, params_, problem_default_values_):
         self.name = 'LeNet5'
 
         # Create the LeNet-5 layers.
-        self.conv1 = torch.nn.Conv2D(1, 6, kernel_size=(5, 5))
```

Comment:
so this is intriguing... I've been running that model several times... are there two version of those functions?

Reply:
Huh. It threw an error for me, which made me look up PyTorch documentation. Doc has the d lower-case, and when I made the change the model ran fine.

Suspicious.